<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentDialog</class>
 <widget class="QDialog" name="AddStudentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Add a new scientist</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>NAME</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_add_scientist_name">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_add_scientist_name"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>GENDER</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_radio_add_scientist_male">
       <property name="text">
        <string>Male</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_radio_add_scientist_femail">
       <property name="text">
        <string>Female</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_add_scientist_gender">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>YEAR BORN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_add_scientist_year_born">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_add_scientist_year_born"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>YEAR OF DEATH</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="button_radio_add_scientist_still_alive">
       <property name="text">
        <string>Still alive</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_error_add_scientist_year_of_death">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="input_add_scientist_year_of_death"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="button_add_scientist_save">
       <property name="text">
        <string>Save scientist</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_add_scientist_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
